homeassistant/components/zha/alarm_control_panel.py

```diff
@@ -28,7 +28,7 @@
 from .core.cluster_handlers.security import (
     SIGNAL_ALARM_TRIGGERED,
     SIGNAL_ARMED_STATE_CHANGED,
-    IasAce as AceClusterHandler,
+    IasAceClusterHandler as AceClusterHandler,
```

Comment:
It's only used once (line 99), but the `as AceClusterHandler` part is probably no longer required now.